table/tables/tables.go

```diff
@@ -1592,6 +1595,10 @@ func shouldWriteBinlog(ctx sessionctx.Context, tblInfo *model.TableInfo) bool {
 	return !ctx.GetSessionVars().InRestrictedSQL
 }
 
+func shouldIncreaseTTLCount(tblInfo *model.TableInfo) bool {
```

Comment:
Is it better to rename the method to `shouldIncreaseTTLMetricCount` ?

Reply:
Yes.